<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWindow</class>
 <widget class="QMainWindow" name="SelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selection des Formations</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-image: url(:/assets/fst.jpeg);
        background-repeat: no-repeat;
        background-position: center;
    }
    QWidget#centralwidget {
        background: rgba(41, 128, 185, 0.85);
    }
    #professorsButton, #demandsButton {
        background-color: rgba(255, 255, 255, 0.15);
        color: white;
        border: none;
        border-radius: 8px;
        font-family: 'Segoe UI', Arial;
        font-size: 14px;
        font-weight: bold;
        padding: 8px 15px;
    }
    #professorsButton:hover, #demandsButton:hover {
        background-color: rgba(255, 255, 255, 0.25);
    }
    QLabel {
        color: white;
        font-family: 'Segoe UI', Arial;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topButtonsLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="professorsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: rgba(255, 255, 255, 0.15);
          font-size: 14px;
         </string>
        </property>
        <property name="text">
         <string>Professeurs</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/assets/professor.png</normaloff>:/assets/professor.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="demandsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: rgba(255, 255, 255, 0.15);
          font-size: 14px;
         </string>
        </property>
        <property name="text">
         <string>Demandes</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/assets/document.png</normaloff>:/assets/document.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="topSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
font-weight: bold;
color: white;
background-color: transparent;
padding: 20px;
border-radius: 10px;
font-family: 'Roboto', sans-serif;</string>
      </property>
      <property name="text">
       <string>Sélectionnez une Formation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <spacer name="horizontalSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="masterButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>250</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-radius: 15px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 0.9), 
                               stop:1 rgba(41, 128, 185, 0.7));
    color: white;
    font-size: 16px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial;
    text-align: center;
    padding-bottom: 20px;
}
QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(52, 152, 219, 0.9),
                               stop:1 rgba(52, 152, 219, 0.7));
}
QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 1),
                               stop:1 rgba(41, 128, 185, 0.8));
}</string>
        </property>
        <property name="text">
         <string>Master en Sciences
et Techniques</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="licenceButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>250</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-radius: 15px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 0.85), 
                               stop:1 rgba(41, 128, 185, 0.65));
    color: white;
    font-size: 16px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial;
    text-align: center;
    padding-bottom: 20px;
}
QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(52, 152, 219, 0.85),
                               stop:1 rgba(52, 152, 219, 0.65));
}
QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 0.95),
                               stop:1 rgba(41, 128, 185, 0.75));
}</string>
        </property>
        <property name="text">
         <string>Licence en Sciences
et Techniques</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="engineerButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>250</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: none;
    border-radius: 15px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 0.95), 
                               stop:1 rgba(41, 128, 185, 0.75));
    color: white;
    font-size: 16px;
    font-weight: bold;
    font-family: 'Segoe UI', Arial;
    text-align: center;
    padding-bottom: 20px;
}
QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(52, 152, 219, 0.95),
                               stop:1 rgba(52, 152, 219, 0.75));
}
QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                               stop:0 rgba(41, 128, 185, 1),
                               stop:1 rgba(41, 128, 185, 0.8));
}</string>
        </property>
        <property name="text">
         <string>Cycle d'Ingénieur</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStack"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
